### DISCUSSION TITLE
Is a 24v power supply required?

### DISCUSSION THREAD

**Discussion by tmanternach** [doudar/SmartSpin2k#357] [Q&A] [answered]
I see there is a 24v power supply listed under recommended hardware. I am building the PCB version 2. Can I use a 12v power supply instead? I don't see where anything requires more than 12 volts, but I am probably missing some obvious.

**Comment by doudar** [comment, marked as answer]
Yep! You can use a 12v power supply as well. The stepper may not have as much torque at higher speeds but that's typically not going to make a noticeable difference. 

Good luck!
